### PR TITLE
Add dependabot

### PR DESCRIPTION
**Pull request by alallema**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: "daily"
```
Comment (bidoubiwa) [inline .github/dependabot.yml:6]: In the other repositories we do it weekly. Did you chose daily on purpose ?

Comment (alallema) [reply .github/dependabot.yml:6]: Yes, I think that weekly is more relevant for repo's that are spammed by notifications, but if you think weekly is better I can change it.

Comment (bidoubiwa) [reply .github/dependabot.yml:6]: I don't know if they are a lot of pckages to update here. You know best :)

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥

**Review by brunoocasali — APPROVED** [review]

**Comment by alallema** [comment]
bors merge

**Pull request merged**